.gitignore

```diff
@@ -8,6 +8,7 @@ cscope.po.out
 cscope.out
 smf/manifests/*.xml
 .DS_Store
+.idea
```

Comment:
Files generated by the user’s editor of choice generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig